src/AssetMapper/SassCssCompiler.php

```diff
@@ -12,21 +12,25 @@
 use Symfony\Component\AssetMapper\AssetMapperInterface;
 use Symfony\Component\AssetMapper\Compiler\AssetCompilerInterface;
 use Symfony\Component\AssetMapper\MappedAsset;
+use Symfony\Component\Filesystem\Path;
 use Symfonycasts\SassBundle\SassBuilder;
 
 class SassCssCompiler implements AssetCompilerInterface
 {
     public function __construct(
         private array $scssPaths,
         private string $cssPathDirectory,
+        private string $projectDir,
         private readonly SassBuilder $sassBuilder
     ) {
     }
 
     public function supports(MappedAsset $asset): bool
     {
         foreach ($this->scssPaths as $path) {
-            if (realpath($asset->sourcePath) === realpath($path)) {
+            $absolutePath = Path::isAbsolute($path) ? $path : Path::makeAbsolute($path, $this->projectDir);
```

Comment:
A bit late, but.. 

I think this may be more performant to do the path computation sooner (in the dependency extension?) and probably add a ".scss" test as starting check there, as this method will be called for all the assets of the project.

Reply:
I agree.

My first idea when implementing this fix was to normalize `root_sass` but the the config component lacks the feature I was searching for (kinda like in the options-resolver component). I didn't think to do it manually in the extension load.

Reply:
I can allocate some time for it in a few days, the change is simple. I just need to check how to setup proper integration tests because my committed unit tests will not be enough.

If there is no existing benchmark I don't really have the time to add one.

Reply:
Sure! Please, feel free to give it a time when you have time, PR is warmly welcome!